tests/conftest.py

```diff
@@ -45,13 +45,17 @@ def __init__(self, method, base_url, url, params, json):
             self._url = url
             self._json = json
 
+            # Extract the ID from the URL if it's a specific resource request
+            self._id = None
+            if "/" in url and not url.endswith("s"):
+                self._id = url.split("/")[-1]
+
         @property
         def headers(self):
             if self._page is not None and self._page < 10:
                 params = "&".join([*self.formatted_params, f"page={self._page + 1}"])
-                return {
-                    "link": (f"<{self._base_url}/{self._url}?{params}>; " 'rel="next"')
-                }
+                url = f"/{self._url}" if not self._url.startswith("/") else self._url
+                return {"link": (f"<{self._base_url}/{url}?{params}>; " 'rel="next"')}
```

Comment:
The `/` will be duplicared here, just before `url`

Reply:
Yes. It's a hack because if I don't duplicate the `//` while doing the pagination the Mock Server will start to shorten the url. I didn't look for the root cause yet, but this hack solves the problem